PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/utils/PluginUtil.java

```diff
@@ -430,4 +452,146 @@ public static Image getImage(String imagePath) {
 			return null;
 		}
 	}
+	
+	public static void forceInstallPluginUsingMarketPlaceAsync(String pluginSymbolicName, String marketplaceURL) {
+		Display.getDefault().asyncExec(new Runnable() {
```

Comment:
can be lambda express `asyncExec(() -> forceInstallPluginUsingMarketplace(...))`